src/io/dataset_loader.cpp

```diff
@@ -28,7 +28,7 @@ void DatasetLoader::SetHeader(const char* filename) {
     // get column names
     if (io_config_.has_header) {
       std::string first_line = text_reader.first_line();
-      feature_names_ = Common::Split(first_line.c_str(), "\t ,");
+      feature_names_ = Common::Split(first_line.c_str(), "\t,");
```

Comment:
I think the data format is either csv (split by ',') or tsv (split by '\t'), is whitespace allowed for splitting the columns? whitespace can appear in feature name.